examples/light-switch-app/silabs/include/BindingHandler.h

```diff
@@ -29,5 +29,6 @@ struct BindingCommandData
     chip::EndpointId localEndpointId = 1;
     chip::CommandId commandId;
     chip::ClusterId clusterId;
+    uint8_t level;
```

Comment:
From a high level perspective, you combining data structure that do not go together.
We should have a seperate structure that is has the application level data that is a union for each grouping we need.

Reply:
Added a union for attributes related to LevelControl command